docs-react-native/react-native/docs/android/permissions.md

```diff
@@ -7,7 +7,37 @@ previous: /react-native/docs/android/interaction
 
 # Understanding permissions
 
-On Android, notification permissions are granted by default. However, a user may revoke them later through various means. In addition, a user may manage or revoke permissions at any of three levels; application-wide, channel groups and channels. 
+On Android 13+, notifications are blocked by default and you will need to present a runtime permission dialog to the user. You have to target at least SDK 33 to perform a runtime request on Android 13.+. The permission is always granted for prior versions.
+
+A user may revoke the permissions later through various means. In addition, a user may manage or revoke permissions at any of three levels; application-wide, channel groups and channels.
+
+# Requesting permissions
+
+To get started and request notification permission from your users, call the [`requestPermission`](/react-native/reference/requestpermission)
+method.
+
+If the user declines the permission, Android prevents the permission dialog from being displayed again. This allows
+the users of your application full control of how notifications are handled:
+
+- If the user declines the runtime permission dialog, they must manually allow notifications via the device Settings UI for your application.
+- If the user accepts the runtime permission dialog, notifications will be shown using the settings requested (e.g. with or without sound).
+- If the user declines the runtime permission dialog, then manually enables permissions via the device Settings UI for you application, then manually disables them again, then you can request permissions again via the runtime permission dialog.
```

Comment:
I believe this is repeated from the first point, so can be removed?
```suggestion
```

Reply:
@helenaford thanks for the review

Maybe it can be reworded? It's not exactly the same as the first point. Observe the following:

1. The code calls `requestNotifications` which triggers the runtime permission dialog
2. The user accepts the notification permissions
3. The user will no longer get the runtime permission dialog when the code calls `requestNotifications`
4. The user then decides they don't want notifications any more and disables them via the app settings through the device settings.
5. The next time the user opens the app and the code calls the `requestNotifications` function, it will display the runtime permission dialog again so they will have to now say "Don't allow"
6. All subsequent calls to `requestNotifications` will now not trigger the runtime dialog anymore... UNLESS the user enables and disables the permission again manually through the app device settings

This whole flow is not as straightforward as on iOS where you are only allowed to ask for the permission once and only once, as far as I know this does not work this way on iOS.

Seems that if the user fiddles with the app notification settings via the device settings then that can cause the runtime permission dialog to appear again.

Reply:
Same would go for if the user declines the runtime permission initially but then goes into device settings and enables the notification permissions and then sometime later disables them again. This would allow the runtime permission dialog to appear again.

Reply:
thank you for the explanation, I think it can overcomplicate it by mentioning this scenario. it's going to be very hard to know if the permissions prompt can appear again! 

The easiest thing to do is show it once and if they decline, always send them to the app store.